handbook/engineering/distribution/goals.md

```diff
@@ -4,11 +4,25 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 ## Medium-term goals (~3-6 Months)
 
+### [Improve deployment pipeline UX](https://github.com/orgs/sourcegraph/projects/96)
```

Comment:
Is there a problem statement beyond the negation of the desired outcomes?

Reply:
There is, I forgot to add it. Ill create another PR to include that.